packages/adapter/src/lib/adapter/adapter.ts

```diff
@@ -2503,7 +2503,7 @@ export class AdapterClass extends EventEmitter {
      * @param args - as many arguments as needed, which will be passed to setTimeout
      * @returns timer id
      */
-    setTimeout(cb: unknown, timeout: unknown, ...args: unknown[]): NodeJS.Timeout | void {
+    setTimeout(cb: unknown, timeout: unknown, ...args: unknown[]): ioBroker.Timeout | void {
```

Comment:
```suggestion
    setTimeout(cb: unknown, timeout: unknown, ...args: unknown[]): ioBroker.Timeout | undefined {
```

The typical meaning of `void` is "this might return something, but you shouldn't use it".